[2025-05-22T05:42:15.457+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-22T05:42:15.856+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_user_pipeline.insert_postgres manual__2025-05-22T05:40:44.862630+00:00 [queued]>
[2025-05-22T05:42:15.873+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_user_pipeline.insert_postgres manual__2025-05-22T05:40:44.862630+00:00 [queued]>
[2025-05-22T05:42:15.874+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-05-22T05:42:15.898+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_postgres> on 2025-05-22 05:40:44.862630+00:00
[2025-05-22T05:42:15.909+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=451) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-22T05:42:15.911+0000] {standard_task_runner.py:63} INFO - Started process 452 to run task
[2025-05-22T05:42:15.909+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'random_user_pipeline', 'insert_***', 'manual__2025-05-22T05:40:44.862630+00:00', '--job-id', '386', '--raw', '--subdir', 'DAGS_FOLDER/random_user_pipeline.py', '--cfg-path', '/tmp/tmpwczu6ant']
[2025-05-22T05:42:15.912+0000] {standard_task_runner.py:91} INFO - Job 386: Subtask insert_***
[2025-05-22T05:42:16.954+0000] {task_command.py:426} INFO - Running <TaskInstance: random_user_pipeline.insert_postgres manual__2025-05-22T05:40:44.862630+00:00 [running]> on host cd3a1fee70a8
[2025-05-22T05:42:18.670+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='papala.ramesh@quation.in,smilesoframs@gmail.com,ramesh.rams333@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='random_user_pipeline' AIRFLOW_CTX_TASK_ID='insert_***' AIRFLOW_CTX_EXECUTION_DATE='2025-05-22T05:40:44.862630+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-22T05:40:44.862630+00:00'
[2025-05-22T05:42:18.671+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-22T05:42:25.733+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-22T05:42:25.734+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-22T05:42:26.093+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=random_user_pipeline, task_id=insert_***, run_id=manual__2025-05-22T05:40:44.862630+00:00, execution_date=20250522T054044, start_date=20250522T054215, end_date=20250522T054226
[2025-05-22T05:42:26.166+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-22T05:42:26.493+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py:1297 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2025-05-22T05:42:26.509+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-22T05:42:26.513+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
